docs/api/plms.yaml

```diff
@@ -285,6 +285,926 @@ paths:
+                    parameters:
+                      - in: query
+                        name: count
+                        schema:
+                          type: number
+                        description: Return list of favorite books with given count.
+                    tags:
+                      - Books
+                    responses:
+                      200:
+                        description: Successful Response
+                        content:
+                          application/json:
+                            schema:
+                              $ref: '#/components/schemas/Books'
+                      400:
+                        description: Bad Request
+                        content:
+                          application/json:
+                            schema:
+                              $ref: '#/components/schemas/Error'
+                      401:
+                        description: Unauthorized
+                        content:
+                          application/json:
+                            schema:
+                              $ref: '#/components/schemas/Error'
+                      408:
+                        description: Request Timeout
+                        content:
+                          application/json:
+                            schema:
+                              $ref: '#/components/schemas/Error'
+                      500:
+                        description: Internal Error
+                        content:
+                          application/json:
+                            schema:
+                              $ref: '#/components/schemas/Error'
+  /books/not-placed:
+    get:
+      operationId: getNotPlacedBooks
+      summary: Not Placed Books
+      description: Get not placed books.
+      parameters:
+        - in: query
+          name: count
+          schema:
+            type: number
+          description: Return list of not placed books (empty shelf id) with given count.
+      tags:
+        - Books
+      responses:
+        200:
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Books'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        408:
+          description: Request Timeout
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        500:
+          description: Internal Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /books/last-added:
```

Comment:
Should we use query parameter here instead of a seperate endpoint?

Reply:
with a number? such as lastAdded=10

Reply:
done as above